1.0.1/jessie/build/Dockerfile

```diff
@@ -21,4 +21,9 @@ RUN curl -o /tmp/packagescache.tar.gz http://dist.asp.net/packagecache/aspnetcor
     # restore 1.0.1 ASP.NET packages
     cd /tmp && dotnet restore packagescache.project.json && \
     rm /tmp/packagescache.project.json && \
+    # set up node
+    apt-get -qq update && \
+    apt-get -qqy install node npm && \
```

Comment:
Isn't that the wrong [node](https://packages.debian.org/jessie/node) ?

Here's what the official node docker install does- [Dockerfile](https://github.com/nodejs/docker-node/blob/5ad063e3ba340743e394114d3038d0fd4e0fe570/4.5/wheezy/Dockerfile#L21-L27)